src/apps/LTIMES_NOVIEW.hpp

```diff
@@ -57,14 +57,14 @@ class LTIMES_NOVIEW : public KernelBase
 
   ~LTIMES_NOVIEW();
 
-  Index_type getProblemSize() const override
+  Index_type getProblemSize() const
   {
     return m_num_d * m_num_g * m_num_z;
   }
 
-  Index_type getItsPerRep() const override
+  Index_type getItsPerRep() const
   {
-    return m_num_d * m_num_m * m_num_g * m_num_z ;
+    return getProblemSize();
```

Comment:
Are we not counting num_m in the iterations?

Reply:
Same response as above.